rdf/rdf12/rdf-n-triples/syntax/manifest.ttl

```diff
@@ -168,7 +166,7 @@ trs:ntriples-star-bad-reified-3 rdf:type rdft:TestNTriplesNegativeSyntax ;
    .
 
 trs:ntriples-star-bad-reified-4 rdf:type rdft:TestNTriplesNegativeSyntax ;
-   mf:name      "N-Triples-star - Bad - reified triples whitespace and terms" ;
+   mf:name      "N-Triples-star - Bad - predicate reified triple" ;
```

Comment:
```suggestion
   mf:name      "N-Triples-star — Bad — predicate reified triple" ;
```

Reply:
This is the way the tests (all three generations) are. Developers use hyphen.
Keep it simple in the manifest because of viewing with the wrong character set.

The rest is automatically produced.

(FYI: Chicago style guide prefers no spaces around em-dashes, which then looks bad in this context.)

Reply:
As I recall, Chicago style guide paradoxically and frustratingly *says* to put no spaces around em-dashes, but *does* put spaces around em-dashes. (Certainly multiple style guides do this.)